### DISCUSSION TITLE
Where can I find the URL

### DISCUSSION THREAD

**Discussion by adrianoap** [MobilityData/gbfs#694] [Q&A] [answered]
Hi, 

I want to started contributing for this project **https://github.com/eskerda/pybikes** and I was searching for a source where I could find the URL for APIs that provide GBFS data.
I found this file **https://github.com/MobilityData/gbfs/blob/master/systems.csv** on this repo but it provides only a website URL and not a source where I could retrieve GBFS data. Is there another file that I could fetch this info? 
Thank you very much!

**Comment by davidgamez** [comment, marked as answer]
Hi @adrianoap, You should be able to retrieve the GBFS data using the information from the system.csv file. Please open the file and look at the _Auto-Discovery URL_ column. This column represents each system's [auto-discovery endpoint](https://gbfs.org/specification/reference/#auto-discovery). That would be enough to get the information you are looking for.
